### DISCUSSION TITLE
conda install issue on macOS

### DISCUSSION THREAD

**Discussion by nisargjoshi10** [deepmodeling/deepmd-kit#1486] [General]
I'm wondering why can't I conda install deepmd-kit on macOS Monterey. I'm running `conda create -n deepmd deepmd-kit=*=*cpu libdeepmd=*=*cpu lammps-dp -c https://conda.deepmodeling.org'

and getting `zsh: no matches found: deepmd-kit=*=*cpu`, same for the gpu version.

**Comment by njzjz** [comment]
There are no macOS packages provided at this time.
